### ISSUE TITLE
Small error in Kinetics model (`val` only)

### ISSUE THREAD

**Issue by moscow25** [facebookresearch/SlowFast#128]
Hi! In initializing output heads for Kinetics, it seems the network will add a `self.act()` layer to all non-training models. This includes testing, but also validation.
https://github.com/facebookresearch/SlowFast/blob/master/slowfast/models/head_helper.py#L213

As a result, since this is set to `softmax` -- the model will apply `softmax` to validation outputs when training with `cross_entropy` loss -- all defaults.

This means the Top-1 and Top-5 error for validation report correctly, but validation loss is incorrect. Not a big deal if just loading pre-trained model, but noticeable when training for scratch, or if adding your own output heads to a pre-trained Kinetics model. 

Apologies if I got something wrong, but pretty sure this is what's happening. Love the repo!
